include/AlgTraits.h

```diff
@@ -117,6 +117,153 @@ struct AlgTraitsHexGL {
   static constexpr stk::topology::topology_t baseTopo_ = stk::topology::HEX_8;
 };
 
+struct AlgTraitsEdge_3D
+{
+  static constexpr int nDim_ = 3;
+  static constexpr int nodesPerElement_ = 3;
+  static constexpr int numScsIp_ = 2;
+  static constexpr int numScvIp_ = 0;
```

Comment:
I am not sure things like, i.e., numScvIp_ are required? The only place such variables are used are in the kernels, correct?